modules/810-deckhouse-web/docs/USAGE.md

```diff
@@ -17,3 +19,5 @@ deckhouseWeb: |
     authSignInURL: "https://<applicationDomain>/sign-in"
     authResponseHeaders: "Authorization"
 ```
+
+You can also configure additional parameters. More information about this [in the next chapter](./configuration.html).
```

Comment:
@Zhbert I think it is unnecessary to link to the configuration page here. It is not a guide or something like that.

Reply:
@z9r5 OK.